### ISSUE TITLE
Add more explicit errors to spell.kak

### ISSUE THREAD

**Issue by Delapouite** [mawww/kakoune#2555]
As mentioned on the forum: https://discuss.kakoune.com/t/how-to-use-spell-command/160 `:spell` fails silently when the correct dependencies are not present on the system.

It can be time consuming to debug, so maybe the script should give more feedback about what's going on. An error message `aspell is missing` or something like that.

**Comment by andreyorst** [comment]
Actually `aspell` was installed in my system. But `aspell-en` wasn't. That's what caused the issue for me. However without `aspell` itself `spell` command still fails silently.

**Comment by lenormf** [comment]
#1726

**Comment by larsks** [comment]
This was closed, but #1726 doesn't seem to address it. I just ran into the original problem -- I didn't have `aspell` installed, and `:spell` was failing silently, and I had no idea what was going on -- and was lost and confused until I found [the original post](https://discuss.kakoune.com/t/how-to-use-spell-command/160).

It would really be nice to have some sort of feedback here.

(kakoune version 2024.05.18)
